src/StageConfig_tests.cc

```diff
@@ -0,0 +1,118 @@
+#include <gtest/gtest.h>
+
+#include "StageConfig.h"
+
+#include "agent_tests.h"
+#include "context.h"
+#include "facility_tests.h"
+
+namespace mbmore {
+
+// Benchmarked against mbmore_enrich_compare.ipynb
+// https://github.com/mbmcgarry/data_analysis/tree/master/enrich_calcs
+namespace stageconfig_test {
+// Fixed for a cascade separating out U235 from U238 in UF6 gas
+// Fixed for a cascade separating out U235 from U238 in UF6 gas
+double M = 0.352;   // kg/mol UF6
+double dM = 0.003;  // kg/mol U238 - U235
+double x = 1000;    // Pressure ratio (Glaser)
+
+// General cascade assumptions
+double flow_internal = 2.0;
+double eff = 1.0;
+double cut = 0.5;
+
+// Centrifuge params used in Python test code
+// (based on Glaser SGS 2009 paper)
+double v_a = 485;                                           // m/s
+double height = 0.5;                                        // meters
+double diameter = 0.15;                                     // meters
+double feed_m = 15 * 60 * 60 / ((1e3) * 60 * 60 * 1000.0);  // kg/sec
+double temp = 320.0;                                        // Kelvin
+
+// Cascade params used in Python test code (Enrichment_Calculations.ipynb)
```

Comment:
is `Enrichment_Calculations.ipynb` in the repository? If it isn't, should it be?

Reply:
```suggestion
// Cascade params used in Python test code (Enrichment_Calculations.ipynb)
// found in enrich_calcs repo listed above.
```

I think directing users to the other repo should suffice?

Reply:
maybe adding a hard link to the repo in question would be better.